### ISSUE TITLE
Highlight support in moddingv2/beatmap discussions

### ISSUE THREAD

**Issue by Ephemeralis** [ppy/osu-web#4127]
Currently, there is no easy way to target conversation at a user discussing an issue within the same moddingv2 discussion comment chains.

Chains tend to sometimes diverge, or people pose different points that need 1-1 resolution within the scope of a single comment. At the moment, people achieve this designation by @ tagging the users involved, similar to how github issues work (@Ephemeralis).

Adding this kind of support for moddingv2 would be excellent, especially if it triggered a notification for a highlighted user who was previously involved in the comment chain. Limiting it to this specific context would negate the need for any broad checks for spam since comment chains rarely exceed a few dozen users, tops.

tl;dr:

add @ support for highlighting people in moddingv2, complete with stylized bolding/popout and email notifications
